visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.scss

```diff
@@ -78,7 +78,7 @@ ribbon-bar-component {
 				}
 
 				&.space {
-					width: 2000px;
+					width: 10000px;
```

Comment:
I changed the width from 1000px to 2000px a few months ago to HOTFIX this issue, because `width 100%` didn't really do the job here.

The question is if we invest the time to try to make it work for every screen with a percentage value or if just keep the super high number here?

Reply:
It was me who put the initial 1000px ;) ...because i spent quite some time with the 100% thing, and as you said that did not really work. I don't think invesing more time into this is worth it; if we have screens with a width of over ~11k pixels, we should think about responsive design anyways, because then all of our controls will look ridiculously small anyways...

Reply:
True. If it's not resulting in a weird scaling bug or messing with our performance, we should be fine with this solution.